ingest/build.gradle

```diff
@@ -9,10 +9,23 @@ plugins {
     id "org.springframework.boot"
 }
 
+sourceSets {
+    integrationTest {
+        java {
+            compileClasspath += main.output + test.output
+            runtimeClasspath += main.output + test.output
+            srcDir file("src/itest/java")
+        }
+        resources.srcDir file("src/itest/resources")
```

Comment:
✏️ consistently use `itest` or `integrationTest`
It looks like this in IntelliJ right now:
![image](https://user-images.githubusercontent.com/8041246/61006137-7774cb00-a31e-11e9-83e1-52f7c14a8f47.png)

Reply:
Both the module name and directory are named `itest`, not sure why IntelliJ is doing that. I don't have an issue with the longer name.

Reply:
I prefer itest